### PR TITLE
JBPM-5566: [Stunner] Labels on Sequence Flows

### PR DESCRIPTION
**Pull request by romartin**
Hey @hasys @tiagodolphine 

See https://issues.jboss.org/browse/JBPM-5566

This provides a basic implementation for labels on sequence flows, it can easily be improved but at least we have something for the release.

Please notice I'll be on sick leave for a while, so if any updates or issues appear on this PR, can you please @tiagodolphine  take care of it? You can for example close these PRs and create your owns with the necessary changes... up to you! 

Relates to
https://github.com/kiegroup/lienzo-core/pull/59
https://github.com/kiegroup/lienzo-tests/pull/41

cc @krisv @mdproctor 

Thanks in advance guys!!

### CODE REVIEW

**Comment by romartin** [comment]
Jenkins execute full downstream build

**Comment by tiagodolphine** [comment]
Hi @hasys can you insert the issue you found on this PR, then I can take a look and see if it will be feasible to solve, probably I'll need to create a new branch based on this one, on my fork.

Thanks.

**Comment by hasys** [comment]
Hi @tiagodolphine, yes sure:

- in my opinion labels are not easy to read, probably it will be good to make it more contrast (see picture below)
- Label ignores new lines defined by user and use it's own alignment (it is common for all labels)
- New lines removed in name property of Sequence flow disappear entirely after save and reopen (it is Sequence flow specific)
- When you have perpendicular angel for Sequence flow label still trying to align it, but in my opinion it's better to display it as is. For example see image below.

image

Also I found unrelated issue: [JBPM-7715](https://issues.jboss.org/browse/JBPM-7715)

**Comment by hasys** [comment]
@tiagodolphine in general this PR solves an issue, thank you @romartin, probably two issues can be fixed in separate PR: 
- make labels more contrast (@wrnkld we need you opinion here)
- New lines removed in name property of Sequence flow

The rest of issues are not related to this PR in my opinion. We can use this [JBPM-6454](https://issues.jboss.org/browse/JBPM-6454) jira issue to handle the rest of issues.

Now going to check the code.

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-lienzo/src/test/java/org/kie/workbench/common/stunner/client/lienzo/shape/view/wires/ext/WiresConnectorViewExtTest.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+import static org.mockito.Mockito.verify;
+
+@RunWith(LienzoMockitoTestRunner.class)
+public class WiresConnectorViewExtTest {
+
+    private MultiPathDecorator HEAD_DECORATOR;
+    private MultiPathDecorator TAIL_DECORATOR;
+    private Point2DArray POINTS;
+    private PolyLine line;
+    private Layer layer;
```
Comment (hasys) [inline kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-lienzo/src/test/java/org/kie/workbench/common/stunner/client/lienzo/shape/view/wires/ext/WiresConnectorViewExtTest.java]: All those fields can be a local variables.

**Comment by tiagodolphine** [comment]
@hasys I can provide the PRs for cherry-pick to 7.11 do you think it is better to wait for the other improvements (color, alignment) and cherry pick all together?

**Pull request merged**